consvc_shepherd/admin.py

```diff
@@ -137,8 +137,19 @@ class AllocationSettingAdmin(admin.ModelAdmin):
     inlines = [PartnerAllocationInline]
     form = AllocationSettingForm
     actions = [publish_allocation]
+    list_display = ["position", "partner_allocation"]
+
+    def partner_allocation(self, obj) -> str:
+        """Partner allocation summary display column."""
+        result = PartnerAllocation.objects.filter(allocation_position=obj)
+        results = [alloc.to_dict() for alloc in result]
+        results.sort(key=lambda item: item.get("percentage"), reverse=True)  # type: ignore
+        row = ""
+        for item in results:
+            row += f'{item.get("partner", "")}: {item.get("percentage", "0")}% '
+        return row
```

Comment:
Good idea, but I don't think we need `to_dict()` for this. We could simplify this to something like:
```suggestion
 def partner_allocation(self, obj) -> str:
        """Partner allocation summary display column."""
        result = PartnerAllocation.objects.filter(allocation_position=obj).order_by(
            "-percentage"
        )
        row = ""
        for item in result:
            row += f"{item.partner.name}: {item.percentage}% "
        return row
```

Reply:
while we're at it, can we add ordering for `AllocationSettingAdmin` using "position"?

Reply:
Nice, that's a lot cleaner. Thx!

Reply:
Hi @tiftran . I did add one additional fallback to check for `NoneType` in the case that a partner is deleted. It will render 'Deleted Partner' and doesn't crash the application. When testing, if a partner is deleted, the `name` of the formerly allocated partner causes the rendering code to crash, so I just check for that in the loop. See below
```python
    def partner_allocation(self, obj) -> str: 
        """Partner allocation summary display column."""
        result = PartnerAllocation.objects.filter(allocation_position=obj).order_by(
            "-percentage"
        )
        row = ""
        for item in result:
            if not item.partner:
                row += f"Deleted Partner: {item.percentage}% "
            else:
                row += f"{item.partner.name}: {item.percentage}% "
        return row
```

<img width="1129" alt="Screen Shot 2023-06-16 at 3 05 30 PM" src="https://github.com/mozilla-services/consvc-shepherd/assets/35045299/bf6d29ee-8c94-41c5-a9e7-559f08ae0bc0">
<img width="983" alt="Screen Shot 2023-06-15 at 11 25 42 PM" src="https://github.com/mozilla-services/consvc-shepherd/assets/35045299/79d1915f-ff7a-4219-996f-b8751b13061b">

Reply:
good catch